src/styles/index.scss

```diff
@@ -1,3 +1,127 @@
+:root {
+  --main-color: #00acdc;
+  --text-accent-color: #060b35;
+  --text-secondary-color: #616070;
+}
+
+html {
+  color: var(--text-secondary-color);
+  font-family: "Roboto", Helvetica, sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
```

Comment:
redundant cause it's default value

Reply:
https://github.com/mate-academy/layout_product-cards/pull/3609/files#r1321869488
you don't need to write font-weight: 400 because it is by default